app/lib/params_validator.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+AVAILABLE_STOCKS = %w[GPW NC].freeze
+AVAILABLE_CONDITION = %w[company_active].freeze
```

Comment:
You have a list of conditions (there is a single element in the list, but it is still a list) so it should be plural:

```suggestion
AVAILABLE_CONDITIONS = %w[company_active].freeze
```

Reply:
Yes, correct, a small detail but essential